content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+    deps: bool = False,
+) -> Dict:
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves the <abbr>params</abbr> tracked in a <abbr>DVC repository</abbr>.
+
+Without arguments, this function will retrieve all <abbr>params</abbr> from all
+parameter files, for the current revision.
+
+See the options below to restrict the <abbr>params</abbr> retrieved.
+
+## Parameters
+
+- **`*targets`** (str, optional): Names of the parameter files to retrieve
+  <abbr>params</abbr> from. If no `targets` are provided, all parameter files
+  will be used. Note that targets don't necessarily have to be defined in
+  `dvc.yaml`
+
+- **`repo`** (str, optional): location of the <abbr>DVC repository</abbr>.
+  Defaults to the current project (found by walking up from thecurrent working
+  directory tree). It can be a URL or a file system path. Both HTTP and SSH
+  protocols are supported for online Git repos (e.g. [user@]server:project.git).
+
+- **`stages`** (Union[str, Iterable[str]], optional): Name(s) of the stages to
+  retrieve <abbr>params</abbr> from. Defaults to `None`. If no stages are
+  provided, all parameters from all stages will be retrieved.
+
+- **`rev`**: (str, optional): Name of the Git
+  [revision](https://git-scm.com/docs/revisions) to retrieve <abbr>params</abbr>
+  from. Example of git revision can be a branch or tag name, a commit hash or a
+  dvc experiment name. Defaults to `HEAD`. If `repo` is not a Git repo, this
+  option is ignored.
+
+- **`deps`**: (bool, optional): Whether to retrieve only <abbr>params</abbr>
+  that are <abbr>stage</abbr> dependencies or not. Defaults to `False`.
+
+## Example: No arguments.
+
+> Working on https://github.com/iterative/example-get-started
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params()
+print(json.dumps(params, indent=4))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  },
+  "featurize": {
+    "max_features": 200,
+    "ngrams": 2
+  },
+  "train": {
+    "seed": 20170428,
+    "n_est": 50,
+    "min_split": 0.01
+  }
+}
+```
+
+## Example: Filtering with `stages`.
+
+> Working on https://github.com/iterative/example-get-started
+
+`stages` can a single string:
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(stages="prepare")
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  }
+}
+```
+
+Or an iterable of strings:
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(stages=["prepare", "train"])
+print(json.dumps(params, indent=2))
+```
+
+```json
+{
+  "prepare": {
+    "split": 0.2,
+    "seed": 20170428
+  },
+  "train": {
+    "seed": 20170428,
+    "n_est": 50,
+    "min_split": 0.01
+  }
+}
+```
+
+## Example: Using `rev`.
+
+> Working on https://github.com/iterative/example-get-started
+
+```py
+import json
+import dvc.api
+params = dvc.api.get_params(rev="tune-hyperparams")
```

Comment:
Needs a bit more explanation I think. Unclear what's happening or why do this from just the command and output samples (which look similar to the other examples).

Reply:
Added an introductory line.